ansible_rulebook/builtin.py

```diff
@@ -841,3 +854,7 @@ def _collect_extra_vars(
         eda_vars["event"] = variables["event"]
     extra_vars[KEY_EDA_VARS] = eda_vars
     return extra_vars
+
+
+def _run_at() -> str:
+    return f"{datetime.utcnow().isoformat()}Z"
```

Comment:
I wonder if this should be instead
```
datetime.now(timezone.utc)
```
Reason: https://aaronoellis.com/articles/python-datetime-utcnow-considered-harmful